src/components/Sensitivity/Sensitivity.tsx

```diff
@@ -0,0 +1,62 @@
+import { Box, Chip, makeStyles, Typography } from '@material-ui/core';
+import { VerifiedUserOutlined } from '@material-ui/icons';
+import { FC } from 'react';
+import * as React from 'react';
+import { Sensitivity as SensitivityType } from '../../api';
+
+const commonChipStyles = {
+  borderRadius: '4px',
+  color: 'white',
+};
+
+const useStyles = makeStyles(
+  ({ palette: { secondary, warning }, spacing }) => ({
+    icon: {
+      height: spacing(2),
+      width: spacing(2),
+      marginRight: spacing(1),
+    },
+    iconWrapper: {
+      display: 'flex',
+      alignItems: 'center',
+      marginBottom: spacing(),
+    },
+    chipLow: {
+      ...commonChipStyles,
+      backgroundColor: '#8f928b',
+    },
+    chipMedium: {
+      ...commonChipStyles,
+      backgroundColor: warning.main,
+    },
+    chipHigh: {
+      ...commonChipStyles,
+      backgroundColor: secondary.main,
+    },
+  })
+);
+
+export interface SensitivityProps {
+  value: SensitivityType;
+}
+
+export const Sensitivity: FC<SensitivityProps> = ({ value }) => {
+  const classes = useStyles();
+
+  const chipClass =
+    value === 'High'
+      ? classes.chipHigh
+      : value === 'Medium'
+      ? classes.chipMedium
+      : classes.chipLow;
```

Comment:
There's a handy library that makes this kinda thing easier. 
```suggestion
  const chipClass = clsx({
    [classes.chip]: true,
    [classes.chipHigh]: value === 'High',
    [classes.chipMedium]: value === 'Medium',
    [classes.chipLow]: value === 'Low',
  });
```
Hmm same amount of lines. But another point I wanted to make is that the common chip class can be still be separated and multiple classes used.

Reply:
Cool - I’ve this with the classNames package before - seems similar. Will add!